### PR TITLE
Register new stable branch of OVS CNI 0.29

### PR DESCRIPTION
**Pull request by phoracek**

### CODE REVIEW

File: github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits-0.29.yaml
Comment (qinqon) [inline github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits-0.29.yaml:4]: We need to state the version at job name to prevent prow from getting confuse

**Review by qinqon — CHANGES_REQUESTED** [review]

File: github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits-0.29.yaml
Comment (phoracek) [reply github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits-0.29.yaml:4]: Ah, good catch. Should be fixed now

**Review by dhiller — APPROVED** [review]
/approve

File: github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits-0.29.yaml
Comment (dhiller) [reply github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits-0.29.yaml]: suggestion: you can add `context: pull-e2e-ovs-cni` here to map to the same GitHub status

Comment (phoracek) [reply github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits-0.29.yaml]: Aha! I did not know. Let me add it. Is it ok to add it to the main branch too? (So I would copy-paste it while branching some future release)

Comment (dhiller) [reply github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits-0.29.yaml]: If you are using the [config-forker from `kubernetes/test-infra`](https://github.com/kubernetes/test-infra/tree/master/releng/config-forker#readme) it will add the context automatically IIRC.

File: github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits.yaml
Comment (dhiller) [inline github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits.yaml:5]: This is not required since it's the default. Also people might think that they need to adjust the context also, no? Maybe you can add it to the file for the older release branch instead, since I think it's missing there.

Comment (dhiller) [reply github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits.yaml:5]: https://github.com/kubernetes/test-infra/blob/f6c02074bb4274351972b7313f64de61ab7bcd87/prow/config/jobs.go#L323

Comment (phoracek) [reply github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits.yaml]: I see your point. I added a little comment to that line and added context to 0.19 too

File: github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits-0.29.yaml
Comment (phoracek) [reply github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits-0.29.yaml]: Gotcha. I would usually just copy the file and change a few fields than try to recall how to use the forker :grimacing:

File: github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits.yaml
Comment (phoracek) [reply github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits.yaml]: @dhiller I can remove it completely if you mind the context on main, your call

File: github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits-0.29.yaml
Comment (dhiller) [reply github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits-0.29.yaml]: > Gotcha. I would usually just copy the file and change a few fields than try to recall how to use the forker grimacing

TBH if we hadn't automated this, I'd be doing it the same way :)

File: github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits.yaml
Comment (dhiller) [reply github/ci/prow-deploy/files/jobs/k8snetworkplumbingwg/ovs-cni/ovs-cni-presubmits.yaml]: I think we are good. In your case using the configforker sounds like overkill.

**Review by dhiller — APPROVED** [review]

**Comment by phoracek** [comment]
Thanks!

/hold cancel

**Pull request merged**
